{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/core/views.py

```diff
@@ -16,9 +21,7 @@
 
 from .models import User
 from .permissions import CreateOnlyPermissions
-{% if cookiecutter.use_graphql == 'n' -%}from .serializers import UserLoginSerializer, UserRegistrationSerializer, UserSerializer{% else %}
-from .serializers import UserLoginSerializer, UserSerializer
-{% endif %}
+from .serializers import UserLoginSerializer, UserRegistrationSerializer, UserSerializer
```

Comment:
Found and fixed a bunch of linting issues while working this PR.
Because of this PR, this should be the LAST time we play the tennis match of linter errors.